google/cloud/storage/bucket.py

```diff
@@ -1021,6 +1023,35 @@ def list_notifications(self, client=None, timeout=_DEFAULT_TIMEOUT):
         iterator.bucket = self
         return iterator
 
+    def get_notification(self, notification_id, client=None, timeout=_DEFAULT_TIMEOUT):
+        """get Pub / Sub notification for this bucket.
+
+        See:
+        https://cloud.google.com/storage/docs/json_api/v1/notifications/get
+
+        If :attr:`user_project` is set, bills the API request to that project.
+
+        :type notification_id: str
+        :param notification_id: The notification id to retrieve the notification configuration.
+
+        :type client: :class:`~google.cloud.storage.client.Client` or
+                      ``NoneType``
+        :param client: Optional. The client to use.  If not passed, falls back
```

Comment:
`Optional.` ==> `(optional)`
Or, at least, make it consistent. The original code shows bad example.

Reply:
Yes, changed it but don't know why whole file have  the inconsistent  format ,like timeout with (optional) and others follow Optional. It needs to be clean